modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/SampleData_conf.pm

```diff
@@ -27,196 +27,199 @@ Questions may also be sent to the Ensembl help desk at
 
 Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_conf
 
-=head1 SYNOPSIS
-
-
 =head1 DESCRIPTION
 
-Pipeline will update sample data in the core database of new/updated species with gene/transcript
-found using compara gene trees, xrefs and supporting evidences.
+Create sample data in the core database of new/updated species,
+using gene trees, xrefs and supporting evidences to find 'good'
+gene/transcript/location examples. Also generate VEP examples.
 
 =cut
 
 package Bio::EnsEMBL::Production::Pipeline::PipeConfig::SampleData_conf;
 
 use strict;
 use warnings;
-use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
-use Bio::EnsEMBL::ApiVersion qw/software_version/;
-use Bio::EnsEMBL::Hive::Version 2.5;
-use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
-
-
-=head2 default_options
-
- Description: It returns a hashref containing the default options for HiveGeneric_conf
- Returntype : Hashref
- Exceptions : None
 
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
 
-=cut
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use Bio::EnsEMBL::Hive::Version 2.5;
 
 sub default_options {
-    my ($self) = @_;
-
-    return {
-        # inherit other stuff from the base class
-        %{ $self->SUPER::default_options() },
-        pipeline_name => 'sample_data_'.$self->o('ensembl_release'),
-        ## 'job_factory' parameters
-        species       => [],
-        antispecies   => [],
-        division      => [],
-        run_all       => 0,
-        meta_filters  => {},
-        history_file => undef,
-        skip_metadata_check => 0,
-        vep_species => $self->o('species'),
-        gene_species => $self->o('species'),
-        vep_division => $self->o('division'),
-        gene_division => $self->o('division'),
-        base_dir => $ENV{'BASE_DIR'},
-        # Only used for vertebrates at the moment
-        maximum_gene_length => 100000,
-    };
+  my ($self) = @_;
+  return {
+    %{$self->SUPER::default_options},
+
+    species      => [],
+    antispecies  => [],
+    division     => [],
+    run_all      => 0,
+    meta_filters => {},
+
+    # A script from the variation repo is needed, so need a path for it.
+    base_dir => $ENV{'BASE_DIR'},
+
+    # Path for small intermediate files
+    tmp_dir => '/hps/nobackup2/production/ensembl/'.$ENV{'USER'}.'/'.$self->o('pipeline_name'),
```

Comment:
Yes; this'll have to change when we move to Codon. But hopefully that will be a little simpler, it can go in `/hps/scratch/$USER`, because there's not much data, and it'll get automatically removed after a week.